.travis.yml

```diff
@@ -28,8 +50,9 @@ before_install:
   - travis_retry composer self-update
 
 before_script:
-  - travis_retry composer install --no-interaction
+  - if [[ $DEPS == 'latest' ]]; then travis_retry composer update --no-interaction --prefer-source ; fi
```

Comment:
With my first comment, this [the two added lines] can be simplified to `composer update --no-interaction --prefer-source $COMPOSER_FLAGS`